Source/control.cpp

```diff
@@ -1176,64 +1176,58 @@ void UpdateManaFlask()
 
 void InitControlPan()
 {
-	size_t v0;         // esi
-	void *v1;          // ecx
-	void *v2;          // ecx
-	void *v3;          // ecx
-	unsigned char *v5; // eax
-
-	v0 = 144 * 640;
-	if (gbMaxPlayers != 1)
-		v0 = 288 * 640;
-	pBtmBuff = DiabloAllocPtr(v0);
-	memset(pBtmBuff, 0, v0);
-	pManaBuff = DiabloAllocPtr(0x1E40);
-	memset(pManaBuff, 0, 0x1E40);
-	pLifeBuff = DiabloAllocPtr(0x1E40);
-	memset(pLifeBuff, 0, 0x1E40);
+	int i;
+
+	if (gbMaxPlayers == 1) {
+		pBtmBuff = DiabloAllocPtr(144 * 640);
+		memset(pBtmBuff, 0, 144 * 640);
+	} else {
+		pBtmBuff = DiabloAllocPtr(288 * 640);
+		memset(pBtmBuff, 0, 288 * 640);
+	}
+	pManaBuff = DiabloAllocPtr(88 * 88);
+	memset(pManaBuff, 0, 88 * 88);
+	pLifeBuff = DiabloAllocPtr(88 * 88);
+	memset(pLifeBuff, 0, 88 * 88);

```

Comment:
Why use particularly `88` here?

Reply:
The graphics is 88x88px. Much more readable then 7744, especially if we where to update it :)

Reply:
Wouldn't a defined named constant be better somewhere, then its even easier to change, you just change it in the one place.

Reply:
If we decide to go down that route we should probably define the size of all sprites

Reply:
Eventually everything will be added.